src/cdk/testing/tests/protractor.e2e.spec.ts

```diff
@@ -197,6 +197,17 @@ describe('ProtractorHarnessEnvironment', () => {
       expect(await input.getProperty('value')).toBe('');
     });
 
+    it('sendKeys method should not focus element if no keys have been specified', async () => {
+      const input = await harness.input();
+      expect(await input.isFocused()).toBe(false);
+      await input.sendKeys();
+      expect(await input.isFocused()).toBe(false);
+      await input.sendKeys('');
+      expect(await input.isFocused()).toBe(false);
+      await input.sendKeys('', '');
+      expect(await input.isFocused()).toBe(false);
+    });
```

Comment:
I decided to not write a test like `should not throw if empty keys have been specified` because this test already covers this. Happy to add an explicit one if desired.